Framework/Algorithms/src/StripPeaks.cpp

```diff
@@ -187,9 +185,10 @@ StripPeaks::removePeaks(API::MatrixWorkspace_const_sptr input,
   prg = 0.3;
```

Comment:
The combination of the `isHistogramData` flag two lines above together with this:
https://github.com/mantidproject/mantid/pull/17004/files#diff-dbacdcd002cca75b9a2d1ad07830e4c4R259

.. can be replaced by `outputWS->points(index)`. Instead of getting `X` which may be bin centers (points or bin edges, you can simply get points directly.